code/modules/recycling/disposal.dm

```diff
@@ -1219,10 +1215,13 @@
 	icon_state = "pipe-j1s"
 	desc = "An underfloor disposal pipe with a package sorting mechanism."
 
-	var/posdir = 0
-	var/negdir = 0
 	var/sortdir = 0
 
+	var/last_sort = FALSE
+	var/sort_scan = TRUE
+	var/panel_open = FALSE
+	var/datum/wires/wires = null // ...Why isnt this defined on /atom...
```

Comment:
Needs to have a qdel_null(wires) in /obj/structure/disposalpipe/sortjunction/Destroy() so we don't hard del